astropy/utils/iers/iers.py

```diff
@@ -800,3 +801,50 @@ def _substitute_iers_b(cls, table):
             table['PM_Y_B'][:n_iers_b] = iers_b['PM_y']
 
         return table
+
+
+class earth_rotation_table(ScienceState):
```

Comment:
Might it be worth overriding the `__repr__` so it does not try to print the whole IERS table?  That isn't really helping out, and basically `<ScienceState earth_rotation_table: IERS_A>` is what users would want to see.

Reply:
Good idea. Since for that it is by far the easiest to adjust `ScienceState` as well, let me also ping @astrofrog (@astrofrog, I changed it such that the context repr never is more than 1 line)

Reply:
@astrofrog - see https://github.com/astropy/astropy/pull/9244/commits/a0b46b0f3a3159915a7d25e03d684c9de928ec88